recipes-devtools/clang/clang_git.bb

```diff
@@ -484,9 +484,12 @@ SYSROOT_DIRS:append:class-target = " ${nonarch_libdir}"
 SYSROOT_PREPROCESS_FUNCS:append:class-target = " clang_sysroot_preprocess"
 
 clang_sysroot_preprocess() {
-	install -d ${SYSROOT_DESTDIR}${bindir_crossscripts}/
-	install -m 0755 ${S}/llvm/tools/llvm-config/llvm-config ${SYSROOT_DESTDIR}${bindir_crossscripts}/
-	ln -sf llvm-config ${SYSROOT_DESTDIR}${bindir_crossscripts}/llvm-config${PV}
+	if ${@ 'true' if 'llvm' in d.getVar('PROVIDES') or '' else 'false' } ; then
+		install -d ${SYSROOT_DESTDIR}${bindir_crossscripts}/
+		install -m 0755 ${S}/llvm/tools/llvm-config/llvm-config ${SYSROOT_DESTDIR}${bindir_crossscripts}/
+		ln -sf llvm-config ${SYSROOT_DESTDIR}${bindir_crossscripts}/llvm-config${PV}
```

Comment:
this is also going to be problematic when llvm bits are provided by clang-native

Reply:
But there is a PROVIDES check. Isn't it enough?